segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/OperationProcessor.java

```diff
@@ -329,6 +329,9 @@ private void processOperations(Queue<CompletableOperation> operations) {
                     CompletableOperation o = operations.poll();
                     this.metrics.operationQueueWaitTime(o.getTimer().getElapsedMillis());
                     try {
+                        if (o.getOperation().getDesiredPriority() != null && o.getOperation().getDesiredPriority().isThrottlingExempt()) {
```

Comment:
Looks good to me.
Just to avoid some unexpected weirdness/resetting of value on calling getElapsedMillis twice, 
let's just capture it once like below.

```
val elapsed = o.getTimer().getElapsedMillis();
this.metrics.operationQueueWaitTime(elapsed );
   try {
           if (o.getOperation().getDesiredPriority() != null && o.getOperation().getDesiredPriority().isThrottlingExempt()) {
                               log.debug("queue wait time for critical operation {} is {}", elapsed);
```

Reply:
Checked the timer logic, it will not reset, but it is a good idea to call it once.